.github/workflows/ci.yml

```diff
@@ -30,7 +30,6 @@ jobs:
           - al2-x64
           - manylinux2014-aarch64
           - manylinux2014-x64
-          - manylinux2014-x86
```

Comment:
ooc, why are we removing this?

Reply:
linux x86 builds of node don't exist from 14 on